src/main/java/mate/academy/hibernate/relations/util/HibernateUtil.java

```diff
@@ -1,9 +1,25 @@
 package mate.academy.hibernate.relations.util;
 
+import mate.academy.hibernate.relations.exception.DataProcessingException;
 import org.hibernate.SessionFactory;
+import org.hibernate.cfg.Configuration;
 
 public class HibernateUtil {
+    private static final SessionFactory sessionFactory = initSessionFactory();
+
+    private HibernateUtil() {
```

Comment:
This private default constructor is correctly added to prevent creating `HibernateUtil` objects, as per the checklist.